Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -307,14 +343,25 @@
 {
 }
 
-bool UnifiedPDFPlugin::handleMouseEvent(const WebMouseEvent&)
+void UnifiedPDFPlugin::updateScrollbars()
 {
-    return false;
+    PDFPluginBase::updateScrollbars();
+
+    CheckedPtr page = this->page();
+    if (!page)
+        return;
+    auto& scrollingCoordinator = *page->scrollingCoordinator();
+    scrollingCoordinator.setScrollingNodeScrollableAreaGeometry(m_scrollingNodeID, *this);
```

Comment:
It's weird for this to happen in a function called `updateScrollbars`. I'd expect an `updateGeometry` or `sizeDidChange` function to be doing this.